provision/static/scenario.yml

```diff
@@ -5,7 +5,7 @@ description: define a static resource to be used throughout teflo
 
 provision:
   - name: static-node
-    role: node
+    groups: node
```

Comment:
You need to fix this in a separate PR, as this is not related to the teflo_init

Reply:
@JunqiZhang0 , this example within the teflo_init directory, so i think we dont need to have a separate PR for this. We can put a new PR , for correcting the original example, if thats what you are talking about. The changes here are only within the teflo_init directory

Reply:
I will correct the original files in a separate PR.
this file is from the original files, so it will be handle in the new PR.

Reply:
This is the new PR  to correct the original files
https://github.com/RedHatQE/teflo_examples/pull/5